jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
-
-1. Claim a namespace (assuming you don't yet have one), using the command `circleci namespace create <NAMESPACEID> <VCS> <YOURORG>`. For example:
-`circleci namespace create sandbox github CircleCI-Public`
+### Orb Publishing Process
 
-2. Create the orb inside your namespace, for example:
-`circleci orb create sandbox/hello-world`
+Before working with orbs, you may find it helpful to gain a high-level understanding the end-to-end orbs publishing process. The diagram shown below illustrates the orbs publishing process.
+![orbsworkflow-v2 image](  {{ site.baseurl }}/assets/img/docs/orbsworkflow-v2.png)
 
-3. Create the content of your orb in a file. You will generally perform this action in your code editor in a git repo made for your orb. For example, let's assume a file in `/tmp/orb.yml` could be made with a simple orb similar to:
-`echo '{version: "2.1", description: "a sample orb"}' > /tmp/orb.yml`
+#### Steps 1 - 3 - Use the CircleCI CLI
+The first step you must perform in the orbs publishing process is to download and install, update, and then configure the CircleCI CLI for use with CircleCI orbs. Detailed instructions for how to install, update and configure the CLI can be found in the [Get the new CircleCI CLI](https://circleci.com/docs/2.0/creating-orbs/#get-the-new-circleci-cli) section on this page.
 
-4. Validate that your code is a valid orb using the CLI. For example, using the path above you could use:
-`circleci orb validate /tmp/orb.yml`
+#### Step 4 - Validate your build configuration
+Once you have configured the CircleCI CLI, you then need to validate your build configuration to ensure you can correctly create or update an orb. You can find detailed instructions on how to validate your build configuration in the [Validating a Build Config] (https://circleci.com/docs/2.0/creating-orbs/#validating-a-build-config) section.
 
-5. Publish a development version of your orb, for example:
-`circleci orb publish /tmp/orb.yml sandbox/hello-world@dev:first`
+#### Step 5 - Set the Version Property to 2.1
+After validating your build configuration, set the version property to 2.1. More information on how to set the version property can be found in the [Set the Version Property to 2.1](https://circleci.com/docs/2.0/creating-orbs/#set-the-version-property-to-21) section on this page.
 
-6. Once you are ready to push your orb to production, you can publish it manually using ```circleci orb publish``` command or promote it directly from the development version. To increment the new dev version to become 0.0.1, use the following command:
-`circleci orb publish promote sandbox/hello-world@dev:first patch`
+#### Step 6 - Create a new Orb using inline template
+CircleCI provides you an example inline template that you can, and should, use when creating your own orbs. The example inline teamplat can be found in the [Example Inlline Template](https://circleci.com/docs/2.0/creating-orbs/#example-inline-template) section on this page.
```

Comment:
It's important to call out here than inline orbs are the easiest way to get started with orbs, because you can reference them from your existing config. For most of our CircleCI orbs, we don't actually inline the orbs because they're not being used within the same yaml file. (An orb can have inline orbs, it's all infinitely composable... right until we cap you at a depth of 100 nested orbs or so.)

`teamplat` -> `template`

> Use our example inline template to get started creating your own orbs. Using inline orbs allows you to immediately reference your work in the current file, without publishing them. This makes it easier to rapidly compile your changes to ensure they behave as-expected.

Reply:
+1 that we should be careful about saying Inline orbs are an intrinsic part of orb publishing -- they are a reasonable strategy to get started in orb authoring, but they are not a necessary part of that process.

Reply:
Re-worded this step to mention that using inline orbs is the easiest way, but not the only way, to author an orb.